.github/workflows/test-snap-can-build.yml

```diff
@@ -0,0 +1,25 @@
+name: 🧪 Test snap can be built on x86_64
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: snapcore/action-build@v1
+        id: build
+
+      - uses: diddlesnaps/snapcraft-review-action@v1
+        with:
+          snap: ${{ steps.build.outputs.snap }}
+          isClassic: 'false'
+          # Plugs and Slots declarations to override default denial (requires store assertion to publish)
```

Comment:
@diddledani What is this used for? Is it required to use the action?

Reply:
The review bit isn't required, but it's a nice sanity check to verify that when we upload to the store it won't get flagged by the review tools there

Reply:
Aha, so it checks whether an upload would trigger a "manual review" when you upload it to your store, and you can manually add plug and slot declarations to override what would trigger a "manual review"?

Reply:
yup :-)

e.g. if you plugged `docker-support`, which would trigger a manual review without an assertion in the store, then you can add a declaration file here to tell review tools that you're allowed to do that

Reply:
for some desktop snaps, the likelihood is you'll want it to state that you're allowed to slot a dbus name for the app.